.github/workflows/test.yaml

```diff
@@ -635,29 +635,30 @@ jobs:
           change, category:performance, category:bugfix, category:documentation, category:internal
         mode: exactly
   docs_only_check:
-    if: ${{ github.repository_owner == 'pantsbuild' }}
+    if: github.repository_owner == 'pantsbuild'
     name: Check for docs-only change
     outputs:
       docs_only: ${{ steps.docs_only_check.outputs.docs_only }}
     runs-on:
     - ubuntu-20.04
     steps:
-    - continue-on-error: true
-      id: files
+    - name: Check out code
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 10
+    - id: files
       name: Get changed files
-      uses: jitterbit/get-changed-files@v1
+      uses: tj-actions/changed-files@v23.1
       with:
-        format: json
+        files_ignore: docs/**
+        files_ignore_separator: '|'
```

Comment:
This is future-proofing, in case we have other paths to ignore (see the generator script for details).